### DISCUSSION TITLE
Foreground color changing

### DISCUSSION THREAD

**Discussion by Dmitriy147** [fyne-io/fyne#4077] [Q&A]
Is it any way to change foreground color from code without creating custom theme? Any example is welcome...

**Comment by andydotxyz** [comment]
You might need to be more specific about what you are trying to do.

Because the foreground colour is defined by the theme you cannot change it without a custom theme.

**Reply by Dmitriy147** [reply]
I need to change only one parametr of the current theme..for example,  something like it realised for switching dark/light theme, scale and changing font:
fyne.Currentapp().Settings().Foregraundcolor.....
or
os.Setenv(""FYNE_FOREGROUNDCOLOR", Green")

**Reply by Jacalz** [reply]
Have you looked at the documentation over at https://developer.fyne.io/extend/custom-theme? I think it shows you how to change specific parts of the theme if I remember correctly.

**Reply by Dmitriy147** [reply]
Ok! I wil ivestigate it!
